package-lock.json

```diff
@@ -1,16 +1,17 @@
 {
-  "name": "my",
+  "name": "do-the-math",
```

Comment:
On startup, your application fails to run when I run `npm run start` I get errors. It worked after I deleted the `package-lock.json` file and the `node_modules` and ran `npm install` again.

- Possible reasons could be that you pushed the `node_modules` directory into your GitHub repo. I see that you included `node_modules` in your `.gitignore` file. But somehow it got pushed to your GitHub repo. Kindly remove it from your GitHub and make sure that it doesn't get pushed again.
![image](https://user-images.githubusercontent.com/38088818/165753005-16fd7faf-ed05-488b-876d-8ed95995439e.png)

- When a user clones your repository, they should not have the `node_modules` directory until they install it locally using `npm install`.
- After debugging the issues, kindly include any other steps that need to be taken in the README to make your application run if any.

Reply:
Thank you, I did that.